### PR TITLE
Vijayasree/ refactor: dmt5 update to new show component

### PR DESCRIPTION
**Pull request by vijayasree-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by salarhfz-fs — APPROVED** [review]

File: src/pages/dmt5/components/_side-tab.tsx [hunk 1]
```diff
@@ -110,25 +110,6 @@ const TabList = styled.div<ContentProps>`
...
-        margin-bottom: 2.3rem;
-
-        &:last-child {
-            margin-bottom: 0;
-        }
-    }
-    @media ${device.mobileL} {
-        margin-bottom: 0;
-    }
-`
```
Comment (sean-binary) [inline src/pages/dmt5/components/_side-tab.tsx:130]: Does this not remove the styles that are tied to those original components as well?

Comment (vijayasree-deriv) [reply src/pages/dmt5/components/_side-tab.tsx:130]: yes , I commented it out initially to check if the design is being effected for device dimensions of tabletS and mobileS but did not see a difference. Hence I removed it and directly updated the Mobile and Desktop components.

Comment (sean-binary) [reply src/pages/dmt5/components/_side-tab.tsx:130]: Thanks for the check 👍

**Review by sean-binary — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]
LGTM

**Pull request merged**